third_party/2and3/boto/kms/layer1.pyi

```diff
@@ -0,0 +1,41 @@
+# Stubs for boto.kms.layer1 (Python 3.6)
+#
+# NOTE: This dynamically typed stub was automatically generated by stubgen.
+
+from typing import Any, Dict, List, Mapping, Optional, Type
+from boto.connection import AWSQueryConnection
+
+class KMSConnection(AWSQueryConnection):
+    APIVersion = ...  # type: str
+    DefaultRegionName = ...  # type: str
+    DefaultRegionEndpoint = ...  # type: str
+    ServiceName = ...  # type: str
+    TargetPrefix = ...  # type: str
+    ResponseError = ...  # type: Type[Exception]
+    region = ...  # type: Any
+    def __init__(self, **kwargs) -> None: ...
+    def create_alias(self, alias_name: str, target_key_id: str) -> Optional[Mapping[str, Any]]: ...
```

Comment:
For return types, it's actually better to be precise and write `Dict` if that's what it returns. As a general rule, we should be permissive in argument types and precise in return types. The first part is so that a type checker will allow people to pass in a custom dictionary-like type if they want to, and the second part is so that people who write code that mutates the dictionary returned from `create_alias` don't get type errors.